src/MiddlewareDispatcher.php

```diff
@@ -41,60 +41,70 @@ public function __construct(
 
         $this->container = $container;
 
-        foreach ($middlewares as $middleware) {
-            $this->add($middleware);
+        for ($i = count($middlewares) - 1; $i >= 0; $i--) {
```

Comment:
Why did you change `foreach` to `for`?

Reply:
Hmm... I think this is faster then foreach+array_reverse().

Reply:
1. You don't need `array_reverse` here
1. [Foreach is faster a bit](https://github.com/viktorprogger/foreach-bench) (without `array_reverse`) and more readable: 
![image](https://user-images.githubusercontent.com/7670669/74837964-ab43b480-5333-11ea-92e9-0febdeeb61c7.png)